CoreUtilities/src/au/gov/asd/tac/constellation/utilities/xml/XmlUtilities.java

```diff
@@ -75,8 +78,13 @@ public XmlUtilities(final boolean namespaceAware) {
         documentBuilderFactory = DocumentBuilderFactory.newInstance();
         documentBuilderFactory.setNamespaceAware(namespaceAware);
         transformerFactory = TransformerFactory.newInstance();
-        transformerFactory.setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "");
-        transformerFactory.setAttribute(XMLConstants.ACCESS_EXTERNAL_STYLESHEET, "");
+        try {
+            // Some implemntations wont support these settings
+            transformerFactory.setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "");
+            transformerFactory.setAttribute(XMLConstants.ACCESS_EXTERNAL_STYLESHEET, "");
+        } catch (IllegalArgumentException ex) {
+            // Do nothing - Some implemntations wont support above settings
```

Comment:
ditto. Can you also make the exception final

Reply:
done